### DISCUSSION TITLE
지정한 위치로 자동이동할 때 애니메이션을 바꾸고 싶습니다.

### DISCUSSION THREAD

**Discussion by kkoyo08** [naverz/zepeto-studio-kor#969] [5. Q&A]
지정한 위치로 자동으로 이동할 때 뛰는 애니메이션에서 걷는 애니메이션으로 바꾸고 싶은데 어떻게 작성해야되나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 이동하는 대상이 NPC라면 아래 내용을 참고하여 주세요 https://github.com/naverz/zepeto-studio-kor/discussions/507#discussioncomment-2400682
내가 직접 조작하는 캐릭터라면 에디터의 Run Threshold값을 1로 변경한 후 MoveTo를 하면 걷기 모션으로 이동시킬 수 있습니다.

<img width="366" alt="스크린샷 2022-07-08 오전 10 07 45" src="https://user-images.githubusercontent.com/100118650/177897055-bafec932-f467-4fc4-82af-2b6d1c8a9b9b.png">

다만 이 방법은 세팅이 되어버리기 때문에 추후에는 캐릭터의 모션을 보다 자유자재로 적용할 수 있는 방안을 준비 중에 있습니다.
감사합니다.

**Reply by kkoyo08** [reply]
코드 안에서 Run Threshold 값을 바꾸려면 어떻게 넣어야 하나요?
